src/main/java/seedu/address/model/person/Name.java

```diff
@@ -48,7 +48,7 @@ public String toString() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof Name // instanceof handles nulls
-                && fullName.equals(((Name) other).fullName)); // state check
+                && fullName.equalsIgnoreCase(((Name) other).fullName)); // state check
```

Comment:
thanks for doing the fix as per discussed! This would fix the issue of duplicate names that are case sensitive.